Gemfile

```diff
@@ -36,6 +36,9 @@ gem "jwt"
 gem "blueprinter"
 
 group :development, :test do
+  # Call 'byebug' anywhere in the code to stop execution and get a debugger console
+  gem "bullet"
+  gem "byebug", platforms: %i[mri mingw x64_mingw]
```

Comment:
We already have pry, do we need byebug too?

Reply:
Came with the merge. will remove.